### PR TITLE
rtmp-services: Add Shareplay TV

### PR DESCRIPTION
**Pull request by vasuais**
### Description
     1. Changes in  services.json file :
          {
             "name": "Shareplay.tv",            
            "more_info_link": "https://shareplay.tv",
            "servers": [
                {
                    "name": "Default",
                    "url": "rtmp://shareplay.tv:833/live"
                }
            ],
            "recommended": {
                "keyint": 2,
                "profile": "main",
                "supported resolutions": [
                    "1920x1080",
                    "1280x720",
                    "852x480",
                    "640x360"
                ],
                "bitrate matrix": [
                    {
                        "res": "640x360",
                        "fps": 30,
                        "max bitrate": 1000
                    },
                    {
                        "res": "640x360",
                        "fps": 60,
                        "max bitrate": 1500
                    },
                    {
                        "res": "852x480",
                        "fps": 30,
                        "max bitrate": 2000
                    },
                    {
                        "res": "852x480",
                        "fps": 60,
                        "max bitrate": 3000
                    },
                    {
                        "res": "1280x720",
                        "fps": 30,
                        "max bitrate": 4000
                    },
                    {
                        "res": "1280x720",
                        "fps": 60,
                        "max bitrate": 6000
                    },
                    {
                        "res": "1920x1080",
                        "fps": 30,
                        "max bitrate": 6000
                    },
                    {
                        "res": "1920x1080",
                        "fps": 60,
                        "max bitrate": 9000
                    }
                ],
                "max fps": 60,
                "max video bitrate": 9000,
                "max audio bitrate": 128
            }
        }

   2. Changes in package.json File:
        {
	"url": "https://obsproject.com/obs2_update/rtmp-services",
	"version": 192,
	"files": [
		{
			"name": "services.json",
			"version": 192
		}
	]
      }

### Motivation and Context
        We are adding a new service provider to your platform. 

### How Has This Been Tested?
        I have updated the services.json and package.json file in my local system and tested my shareplay.tv configuration.
        
       Hardware: HP ProBook (Intel Core i5 8250U)
       OS: Ubuntu 20.04,
       OBS Version: Obs Studio 27.2.1 (64 bit)

### Types of changes
         As a new service provider add on your platform.

### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [ ] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [x] The code has been tested.
- [ ] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by gxalpha** [comment]
As mentioned before, please make sure to fill out the PR template and confirm you are a representative of shareplay.tv
Also, there is no need to close and open a new PR, you can make changes to this one.

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -2515,6 +2515,73 @@
                 "max video bitrate": 7200,
                 "max audio bitrate": 196
             }
+        },
+        {
+            "name": "Shareplay.tv",
+            "common":true,
```
Comment (gxalpha) [inline plugins/rtmp-services/data/services.json:2521]: Please do not set common to true.
The decision to set this to true would be made by the OBS Project.

**Review by gxalpha — CHANGES_REQUESTED** [review]
See the comment below.
Also, make sure to not change the permissions on the files you’re editing (right now, 644 gets changed to 755).
Again, there is no need to close this PR and open a new one, just squash and force push it. In case you’re unfamiliar with these terms, they are git commands - you can read git-scm for more info

File: plugins/rtmp-services/data/services.json (hunk 1 above)
Comment (vasuais) [reply plugins/rtmp-services/data/services.json:2521]: okey

**Comment by vasuais** [comment]
> As mentioned before, please make sure to fill out the PR template and confirm you are a representative of shareplay.tv Also, there is no need to close and open a new PR, you can make changes to this one.

Please give me your PR template  Link

**Comment by gxalpha** [comment]
In the PR head there are fields where you should enter details, like Description:
https://github.com/obsproject/obs-studio/pull/6012#issue-1147706663
Make sure that gets filled out (just edit it)

**Comment by vasuais** [comment]
> In the PR head there are fields where you should enter details, like Description: [#6012 (comment)](https://github.com/obsproject/obs-studio/pull/6012#issue-1147706663) Make sure that gets filled out (just edit it)

I have updated  PR template. Please check

**Comment by gxalpha** [comment]
Thanks.
Are you a representative of Shareplay?

**Comment by vasuais** [comment]
> Thanks. Are you a representative of Shareplay?

yes

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -2515,6 +2515,72 @@
                 "max video bitrate": 7200,
                 "max audio bitrate": 196
             }
+        },
+        {
+            "name": "Shareplay.tv",            
+            "stream_key_link": "https://shareplay.tv/no-stream",
```
Comment (dodgepong) [inline plugins/rtmp-services/data/services.json:2521]: Is this the correct URL whereby users can retrieve their stream key? This doesn't look correct to me at first glance.

Comment (vasuais) [reply plugins/rtmp-services/data/services.json]: I have updated this link. User login on our site and request for the steam key. we have shown the Get Steam key menu for login user

**Comment by vasuais** [comment]
@dodgepong ,
I have updated my code. Please check

File: plugins/rtmp-services/data/services.json (hunk 2 above)
Comment (vasuais) [reply plugins/rtmp-services/data/services.json]: @dodgepong ,
I have updated  services.json file.

**Comment by gxalpha** [comment]
@vasuais please be patient if you don't get a response right away. People might not be available immediately.
Also again, please make sure permissions on the files don't get changed.

File: plugins/rtmp-services/data/services.json (hunk 2 above)
Comment (RytoEX) [reply plugins/rtmp-services/data/services.json]: It looks like you completely removed this link from your additions. Was that intentional?

Comment (vasuais) [reply plugins/rtmp-services/data/services.json]: We have at times one user live stream on Shareplay.  Any user lives stream on shareplay his/her First login and get stream key.  Remove URL is our live stream page URL. So I have removed it.

**Comment by Haris-Solo** [comment]
Thanks to all Team members

**Pull request merged**